docker-compose/kafka.yaml

```diff
@@ -1,9 +1,10 @@
 version: "2.1"
 
 services:
-  zoo:
-    image: confluentinc/cp-zookeeper:7.2.1
-    hostname: zoo
+  zookeeper:
+    image: ghcr.io/trifork/cheetah-kafka:pr-4@sha256:74b32f4649f5bcbc84626787fbcdaf6c817b02ac876bddfd7e67ea4e9c93e340
```

Comment:
This will request them to have run `docker login`

I guess the solution would be to link to:
https://docs.cheetah.trifork.dev/getting-started/guided-tour/prerequisites#run-standard-jobs

in readme.md ?

Reply:
also, can we use normal zookepper or do we have to use the cheetah-kafka version?